docs/content/operations/web-consoles.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: doc_page
+title: "Web Consoles"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+# Web Consoles
```

Comment:
I think we should only talk about the unified console and really de-emphasize the old consoles

Reply:
Hmm, I think we can talk about only the unified console in the future once the router becomes a mandatory module of a druid cluster or the unified console can be run on other modules. But, for now, I guess it's not easy to tell them about only the unified console. Even all our tutorials are saying the old consoles.

How about mentioning the unified console first and then others later in this page? I guess a sort of screenshots of the unified console would help too.

Reply:
I also think operationally the old consoles still serve a purpose in times of outages to help isolate where the issues are happening, since _i think_ the new console requires basically all of the cluster to be up to work. Until either the unified console can more gracefully degrade in functionality, or the old consoles can be replaced with new ui's built with these new ui controls but hosted on their respective services to serve as fallbacks, I think it's worth having them documented.

Reply:
Is the router not going to be packaged by default in the new 0.14 packaging? I would assume if that's the case, then everyone should be using the unified console. Asking ppl to install the router and then use the console will just mean new folks trying Druid may never find it.

Reply:
There are example and quickstart confs for the router now, but there's no guarantee that people would use the router in their prod clusters since it's an optional component, and existing clusters wouldn't necessarily have a router, so I think it needs to be called out here